ihatemoney/models.py

```diff
@@ -257,6 +257,14 @@ def get_member_bills(self, member_id):
             .order_by(Bill.id.desc())
         )
 
+    def get_most_recent_bill(self):
+        """Returns the most recent bill (according to bill date) or None if there are no bills"""
+        return self.get_bills_unordered().order_by(Bill.date.desc()).first()
+
+    def get_least_recent_bill(self):
```

Comment:
Shouldn't we name this `get_oldest_bill`?

Reply:
You're right, I like compliated names apparently :)